async_geoprocess/src/cumulus_geoproc/geoprocess/snodas/interpolate.py

```diff
@@ -8,8 +8,9 @@
 from string import Template
 
 import pkg_resources
+from botocore.errorfactory import ClientError
 from cumulus_geoproc import logger
-from cumulus_geoproc.configurations import CUMULUS_PRODUCTS_BASEKEY
+from cumulus_geoproc.configurations import CUMULUS_PRODUCTS_BASEKEY, ENDPOINT_URL_S3
```

Comment:
import ENDPOINT_URL_S3 import will not be required after addressing comment below.

Reply:
@jeffsuperglide. I think this import is still present in the latest changes, but unused